src/niscope/system_tests/test_system_niscope.py

```diff
@@ -309,7 +309,7 @@ def test_disable(session):
 
 def test_configure_ref_levels(session):
     session.configure_ref_levels()
-    assert 90.0 == session.meas_chan_high_ref_level
+    assert 90.0 == session._meas_chan_high_ref_level
```

Comment:
So we are confident that these attributes are part of the measurement library... ?

Really nice of us to have test coverage.